src/repo.rs

```diff
@@ -1088,6 +1088,21 @@ impl Repository {
         }
     }
 
+    /// Get the blame for a single file.
+    pub fn blame_file(&self, path: &Path, opts: Option<&mut BlameOptions>)
+                                                -> Result<Blame, Error> {
```

Comment:
My current style here is to align the `->` with the first character after the opening `(` of the arguments.